src/Table/Table.js

```diff
@@ -383,26 +290,13 @@ export default class Table extends React.Component {
 
     // Use scroll table on first render to check if we need to scroll
     // and if content is bigger than its container
-    if ((props.useFlex || itemHeight === 0 || itemListHeight > containerHeight)
-      && props.useScrollTable) {
-      tableContent =
-        this.getScrollTable(
-          columns, data, sortBy, itemHeight, containerHeight, idAttribute
-        );
-    } else {
-      tableContent = this.getTable(columns, data, sortBy, idAttribute);
-    }
-
-    let styles = {};
-    let className = '';
-
-    if (props.useFlex) {
-      className = 'no-overflow';
-      styles.flexGrow = 1;
-    }
+    let tableContent =
```

Comment:
Always render `VirtualList` instead of doing mass logic in order to determine which table to use